src/overlays/actors/ovl_En_Elf/z_en_elf.c

```diff
@@ -1083,8 +1083,9 @@ void func_80A0461C(EnElf* this, PlayState* play) {
     } else {
         arrowPointedActor = play->actorCtx.targetCtx.arrowPointedActor;
 
-        // `R_SCENE_CAM_TYPE` is checking for either `SCENE_CAM_TYPE_SHOP`, `SCENE_CAM_TYPE_FIXED`, or
-        // `SCEME_CAM_TYPE_SHOOTING_GALLERY`
+        // `R_SCENE_CAM_TYPE` is not composed of bit flags, but is being checked as if it is,
```

Comment:
period instead of comma
also feel like this sentence could be improved, but I couldnt think of anything else. mainly the "being checked as if it is" part. But maybe its fine

Reply:
I tried rephrasing to emphasize the bit-part of it